src/libical/icalparameter.h

```diff
@@ -615,7 +615,7 @@ LIBICAL_ICAL_EXPORT icalparameter_kind icalparameter_string_to_kind(const char *
 /**
  * @brief Checks the validity of a ::icalparameter_kind
  * @param kind The icalparameter_kind
- * @return 1 if if the kind is valid, 0 otherwise
+ * @return 1 if if @a kind is valid, 0 otherwise
```

Comment:
why @ a ? 
Lol. Why does github capitalize the a?
Ah, A - the user. don't hilight them.

Reply:
Same as above, turns it into a parameter reference. (Or styles it like one, which is all Doxygen can really do)

Reply:
"@return 1 if the kind is valid, 0 otherwise"

Reply:
@winterz is that what you want me to put instead?

Reply:
yes, or "@return 1 if the @a kind is valid, 0 otherwise".
whichever you prefer.

Reply:
Ok, got it.